src/methods/status.rs

```diff
@@ -1,3 +1,28 @@
+//! Queries the status of a given node
```

Comment:
Let's end this with a period (`.`).

And can we try “Requests the status of the connected RPC node” or something to that effect.

Reply:
Yeah, that sounds more accurate